src/commands/moderation/infractions/functions/timeout.ts

```diff
@@ -92,6 +96,12 @@ const logFailed = makeEmbed({
     color: Colors.Red,
 });
 
+const communicationNotDisabledEmbed = (discordUser: User) => makeEmbed({
+    title: 'Timeout - Communication not disabled',
+    description: `Bot has not detected that ${discordUser.toString()} has disabled communication. Timeout may have failed.`,
```

Comment:
```suggestion
    description: `Bot has not detected that ${discordUser.toString()} was successfully timed out. Timeout may have failed.`,
```
Suggest updating the language to be a bit more generic. The concept "CommunicationDisabled" is an internal API term, so for non-Discord-API-savy people, it could be a little confusing. (I was confused :D)